include/event_manager.h

```diff
@@ -30,6 +30,42 @@
 extern "C" {
 #endif
 
+/**
+ * @brief List of bits in event type flags.
+ */
+enum event_type_flags {
+INIT_LOG_ENABLE = 1,
```

Comment:
Consider using a common prefix for the enum. E.g. `EVENT_TYPE_`

Reply:
> I would also add documentation, because this is part of the user API. Inform that HAS_DYNDATA cannot be set by user too (I would even add build assert to ensure that user does not try to set it).

As said in the beginning I will add documentation and update all our apps after after I will be certain that there will be no more changes.

Reply:
If the documentation is missed - this should be draft. I cannot tick it green if we know that there is something missed.

Reply:
Thanks, for reminding about it. Add change in changelog and forgot to do it in documentation :) 
Added it now.